client/app/cloud/project/compute/infrastructure/ip/failover/buy/cloud-project-compute-infrastructure-ip-failover-buy.html

```diff
@@ -86,6 +86,10 @@ <h2 class="oui-modal__title" data-translate="cpciif_buy_title"></h2>
                     <small>* </small><small data-translate="cpciif_buy_button_infos"></small>
                  </div>
             </div>
+            <oui-message data-type="alert"
```

Comment:
According to the documentation `type="alert"` does not exist (ref: https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/message#api).